yarn.lock

```diff
@@ -4962,6 +4962,8 @@ __metadata:
     "@types/progress": ^2.0.3
     "@types/prompts": ^2.0.13
     "@types/yup": ^0.29.11
+    "@yarnpkg/esbuild-plugin-pnp": ^2.0.1-rc.3
```

Comment:
It turns out that I did not need this, it was working without but there might be some edge cases I am not hitting where it is needed.

Reply:
We'll probably need to use this when we move `defineSchema` to `tinacms`, because at that point the function will be coming from another module and esbuild will need to know where to find it. The tricky part will be only including this plugin when the environment is PNP, which hopefully is as easy as checking for one of these  https://yarnpkg.com/advanced/pnpapi#runtime-constants.

Find to leave it/out for now.